assembly/src/parsers/serde/mod.rs

```diff
@@ -124,6 +124,8 @@ pub enum OpCode {
     U32UncheckedMin = 105,
     U32CheckedMax = 106,
     U32UncheckedMax = 107,
+    U32CheckedPopcnt = 229,
+    U32UncheckedPopcnt = 230,
```

Comment:
I'd prefer to keep sequential numbering (i.e., change `U32CheckedPopcnt` to 108,  `U32UncheckedPopcnt` to 109 etc.). I know this will update many lines of code - but I think it is better long-term to do it like this.